pkg/ingress/kube/ingressv1/controller.go

```diff
@@ -507,47 +506,44 @@ func (c *controller) ConvertHTTPRoute(convertOptions *common.ConvertOptions, wra
 		}
 
 		wrapperHttpRoutes := make([]*common.WrapperHTTPRoute, 0, len(rule.HTTP.Paths))
+
 		for _, httpPath := range rule.HTTP.Paths {
+			var pathType common.PathType
```

Comment:
Also should add this support for ingress v1beta1.

Reply:
ok